src/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -691,19 +699,10 @@ public boolean match(final Unit transport) {
       // mixed transport categories, but there is no UI to manage that anyway.
       // Players will need to load incrementally in such cases.
       if (defaultSelections.containsAll(candidateTransports)) {
-        boolean spaceLeft = false;
-        for (final Unit transport : candidateTransports) {
-          final int capacity = availableCapacityMap.getInt(transport);
-          if (capacity >= minTransportCost) {
-            spaceLeft = true;
-            break;
-          }
-        }
-        if (!spaceLeft) {
-          return candidateTransports;
-        }
```

Comment:
This block isn't needed now that the algorithm above finds min number of transports needed. As it doesn't matter if say the last transport has available capacity since we already know all transports are required.